tests/zentorch/testing_utils.py

```diff
@@ -0,0 +1,171 @@
+    "mpt": {"hf-internal-testing/tiny-random-MptForCausalLM": ["text-generation"]},
+    "gpt2": {"hf-internal-testing/tiny-random-GPT2LMHeadModel": ["text-generation"]},
+    "gptj": {"hf-internal-testing/tiny-random-GPTJForCausalLM": ["text-generation"]},
+    "bloom": {"hf-internal-testing/tiny-random-BloomForCausalLM": ["text-generation"]},
+    "llama": {"hf-internal-testing/tiny-random-LlamaForCausalLM": ["text-generation"]},
+    "falcon": {"hf-internal-testing/tiny-random-FalconForCausalLM": ["text-generation"]},
+    "gpt-neo": {"hf-internal-testing/tiny-random-GPTNeoForCausalLM": ["text-generation"]},
+    "mistral": {"hf-internal-testing/tiny-random-MistralForCausalLM": ["text-generation"]},
+    "gpt-neox": {"hf-internal-testing/tiny-random-GPTNeoXForCausalLM": ["text-generation"]},
+    "starcoder2": {"hf-internal-testing/tiny-random-Starcoder2ForCausalLM": ["text-generation"]},
+    "gpt-bigcode": {"hf-internal-testing/tiny-random-GPTBigCodeForCausalLM": ["text-generation"]},
+    # text encoder-decoder
+    "t5": {"hf-internal-testing/tiny-random-t5": ["text2text-generation"]},
+    "bart": {"hf-internal-testing/tiny-random-bart": ["text2text-generation"]},
+    # image encoder text decoder
+    "llava": {"IlyasMoutawwakil/tiny-random-LlavaForConditionalGeneration": ["image-to-text"]},
+    "blip": {
+        "hf-internal-testing/tiny-random-BlipForConditionalGeneration": ["image-to-text", "visual-question-answering"]
+    },
+    "blip2": {
+        "hf-internal-testing/tiny-random-Blip2ForConditionalGeneration": ["image-to-text", "visual-question-answering"]
+    },
+    # automatic speech recognition
+    "whisper": {"openai/whisper-tiny": ["automatic-speech-recognition"]},
+}
+
+SUPPORTED_DIFFUSION_PIPELINES_TINY = {
+    "stable-diffusion": {"hf-internal-testing/tiny-stable-diffusion-torch": ["text-to-image"]},
+    "stable-diffusion-xl": {"hf-internal-testing/tiny-stable-diffusion-xl-pipe": ["text-to-image"]},
+}
+
+
+def load_transformers_model(model_id: str, task: str):
+    if task == "text-classification":
+        return AutoModelForSequenceClassification.from_pretrained(model_id)
+    elif task == "image-classification":
+        return AutoModelForImageClassification.from_pretrained(model_id)
+    elif task == "text-generation":
+        return AutoModelForCausalLM.from_pretrained(model_id)
+    elif task == "text2text-generation":
+        return AutoModelForSeq2SeqLM.from_pretrained(model_id)
+    elif task == "automatic-speech-recognition":
+        return AutoModelForSpeechSeq2Seq.from_pretrained(model_id)
+    elif task == "audio-classification":
+        return AutoModelForAudioClassification.from_pretrained(model_id)
+    elif task == "image-to-text":
+        return AutoModelForVision2Seq.from_pretrained(model_id)
+    elif task == "visual-question-answering":
+        return AutoModelForVisualQuestionAnswering.from_pretrained(model_id)
+    else:
+        raise ValueError(f"Task {task} not supported")
```

Comment:
Why not use `TasksManager.get_model_from_task`?

Reply:
Probably ok to have not have dependency on optimum.exporters unless needed?

Reply:
I think it's fine (especially in tests), tasks.py don't require any dependency apart from transformers/huggingface_hub